vecgeom.spec

```diff
@@ -19,7 +26,13 @@ cmake ../%{n}-%{realversion} \
   -DBACKEND=Scalar \
   -DUSOLIDS=ON \
   -DUSOLIDS_VECGEOM=ON \
+%if %isamd64
   -DVECGEOM_VECTOR=sse3 \
+%else
+  -DVECGEOM_VECTOR=native \
```

Comment:
Do we really want to have this? Native could be problematic if it picks specific CPU/SOC instructions.

Reply:
I didn't want to cut any of the flags as I don't know how it would change the installation. The sse3 was what broke the arm64 IB so that's how we ended up with this PR. It transforms to -msse3 compiler option which the compiler doesn't recognize.